backend/app/Http/Controllers/ProfileController.php

```diff
@@ -32,6 +32,7 @@ public function updateMe()
         $data = json_decode(Request::getContent(), true);
 
         $oldEmail = $user->email;
+        $oldNickName = $user->nickname;
```

Comment:
Where do you use this?

Also, instead of storing old values and comparing against current to see if they changed, lets just use Laravel Eloquent's `->isDirty()` before calling save to see if it has changed: https://laravel.com/api/5.3/Illuminate/Database/Eloquent/Model.html#method_isDirty

Reply:
oh TIL about that method, cool